apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.Tests/AggregationsTest.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System.Collections.Generic;
+using Xunit;
+using static Google.Cloud.Datastore.V1.AggregationQuery.Types;
+using static Google.Cloud.Datastore.V1.Aggregations;
+
+namespace Google.Cloud.Datastore.V1.Tests;
+public class AggregationsTest
+{
+    [Fact]
+    public void AggregationTest()
```

Comment:
This isn't really testing what Average and Sum do at all.

To see what I mean, try change your Average and Sum implementations to do the wrong thing - to ignore the alias, or even make Average use the Sum operator and vice versa. This test is currently just equivalent to:

```csharp
Assert.Equal(Average("age"), Average("age"));
Assert.Equal(Sum("height"), Sum("height"));
```